src/interfaces/form.interface.ts

```diff
@@ -0,0 +1,29 @@
+import { FieldError, UseFormRegister } from "react-hook-form";
+
+export interface FormData {
+  id: string;
+  name: string;
+  subName: string;
+  priority: "Urgent" | "High" | "Normal" | "Low";
```

Comment:
It seems that this is repeated in other places, I would suggest storing this in a separate type.